clients/iota-go/iotajsonrpc/dynamic_field.go

```diff
@@ -8,7 +8,7 @@ import (
 // in iotago/crates/iotago-types/src/dynamic_field.rs
 type DynamicFieldInfo struct {
 	Name       iotago.DynamicFieldName                        `json:"name"`
-	BcsName    iotago.Base58                                  `json:"bcsName"`
```

Comment:
Can the type Base58 be removed if it's unused?

Reply:
Its still required for the Digest. That's the only last thing that's base58